File: src/etch/bindings/csharp/support/_Etch_AuthException.cs
using System;
using System.Text;
using System.Collections.Generic;

namespace Etch.Support
{
    public class _Etch_AuthException : _Etch_RuntimeException
    {
        public _Etch_AuthException()
        {
            // dont initialize anything
        }

        public _Etch_AuthException( String msg ) : base( msg )
        {
        }
    }
}

File: src/etch/bindings/java/support/_Etch_AuthException.java
/* $Id$
 *
 * Created by wert on Jul 24, 2007.
 *
 * Copyright (c) 2007 Cisco Systems, Inc. All rights reserved.
 */

package etch.bindings.java.support;

/**
 * Exception thrown when messages marked with @Authorize fail.
 */
public class _Etch_AuthException extends _Etch_RuntimeException
{
	private static final long serialVersionUID = -5541931265968145750L;

	/**
	 * Constructs the _Etch_AuthException.
	 */
	public _Etch_AuthException()
	{
		// don't init anything.
	}

	/**
	 * Constructs the _Etch_AuthException.
	 * 
	 * @param msg a message describing the failure.
	 */
	public _Etch_AuthException( String msg )
	{
		super( msg );
	}
}
